docs/operate/docker-quickstart.mdx

```diff
@@ -82,18 +82,50 @@ You should change the service name, db path, and config file locations to match
 AWS_ACCESS_KEY_ID=<Your AWS access key ID>
 AWS_SECRET_ACCESS_KEY=<Your AWS secret access key>
 DB_PATH=./ethereum/hyperlane_db
-METRICS_PORT=9090 # Default is 9090, should match port set in config.json
+METRICS_PORT=9090 # external port, default is 9090 and should be different for each validator ie 9091, 9092, ...
 CONFIG_FILE=./ethereum/config.json
 SERVICE_NAME=ethereum
 ```
 
 ## 5. Configure Docker Compose (docker-compose.yml)
 
 ```yaml
```

Comment:
how do you feel about minimising the .env file to just
```env
AWS_ACCESS_KEY_ID=<Your AWS access key ID>
AWS_SECRET_ACCESS_KEY=<Your AWS secret access key>
SERVICE_NAME=ethereum
```

and streamlining the compose file by using the service name to infer the configs?
```yaml
services:
  ethereum:
    <<: *common-validator
    ports:
      - "9091:9090/tcp"

  optimism:
    <<: *common-validator
    ports:
      - "9092:9090/tcp"

  base:
    <<: *common-validator
    ports:
      - "9093:9090/tcp"

x-common-attributes: &common-validator
  image: gcr.io/abacus-labs-dev/hyperlane-agent:7a8478b-20240703-113821
  command: ./validator
  container_name: ${SERVICE_NAME}-validator
  environment:
    CONFIG_FILES: /config.json
    AWS_ACCESS_KEY_ID: ${AWS_ACCESS_KEY_ID}
    AWS_SECRET_ACCESS_KEY: ${AWS_SECRET_ACCESS_KEY}
  configs:
    - source: ./${SERVICE_NAME}/config.json
      target: /config.json
  volumes:
    - ./${SERVICE_NAME}/hyperlane_db:/mnt/hyperlane_db
  restart: unless-stopped
```

Reply:
I just pushed a new commit with a tested dockerfile